lib/screens/transfer/receive/receive_enter_data/interactor/viewmodels/receive_enter_data_state.dart

```diff
@@ -10,12 +12,11 @@ class ReceiveEnterDataState extends Equatable {
   final PageCommand? pageCommand;
   final String? errorMessage;
   final RatesState ratesState;
-  final String fiatAmount;
-  final String seedsAmount;
+  final double fiatAmount;
```

Comment:
seedsAmount was redundant as it's already captured in 'quantity' - this code would be a lot clearer if using our data models here.